projects/core/utils/mask/create-auto-corrected-money-pipe.ts

```diff
@@ -32,6 +33,11 @@ export function tuiCreateAutoCorrectedNumberPipe(
     }
 
     return (conformedValue, config) => {
+        // Removing everything by selecting and pressing '-'
+        if (!conformedValue && config.rawValue === '-' && allowNegative) {
```

Comment:
are there any tests for this case?

Reply:
No, this is just an edge case which we patched. We should focus on finding a better masking solution instead. Something to look forward next year.